scripts/get-mirrors-rss

```diff
@@ -12,10 +12,12 @@ XML_FILE=${PROJECT_DIR}/etc/ubuntu-mirrors-rss.xml
 
 mkdir -p ${PROJECT_DIR}/etc
 rm -f $TEST_FILE
-wget -q -O $TEST_FILE https://launchpad.net/ubuntu/+cdmirrors-rss
 
-RESULT=$(grep "Ubuntu CD Mirrors Status" $TEST_FILE)
+if [ ! -f $XML_FILE ] ; then
```

Comment:
I can see why you did this - you don't want the mirror file to be downloaded every time in local development.

Unfortunately, this won't work for Jenkins. In Jenkins we need the mirrors file to refresh every time we build.

We could solve this by doing `rm etc/ubuntu-mirrors-rss.xml` before `./run build` in Jenkins - but that's a bit of weird structure, because we're expecting Jenkins to have deep knowledge of how this script works, but not letting it run the script directly.

If the time taken to download the mirror file every time is a concern (which I'd say it probably is), then I'd suggest this:

-  We remove the `if [ ! -f £XML_FILE ]` here, so the script downloads the mirror file every time
- We remove `./scripts/get-mirrors-rss` from `build`, and instead put it in a new `package.json` script called `get-mirrors`
- We document that, if you want the mirrors file for local development, you can run manually `./run exec yarn run get-mirrors`
- In the Jenkins build job we run `./run build && ./run exec yarn run get-mirrors`

Wadayathink?

Reply:
Aye sink it maykes cense